src/core_plugins/kbn_vislib_vis_types/public/heatmap.js

```diff
@@ -1,23 +1,23 @@
-import { VisVisTypeProvider } from 'ui/vis/vis_type';
```

Comment:
not quite sure how I triggered it, but ran into the following error when creating heatmap charts.

```
heatmap_chart.js:179 Uncaught (in promise) TypeError: xScale.rangeBand is not a function
    at HeatmapChart.addSquares (heatmap_chart.js:179)
    at SVGSVGElement.<anonymous> (heatmap_chart.js:271)
    at d3.js:962
    at d3_selection_each (d3.js:968)
    at Array.d3_selectionPrototype.each (d3.js:961)
    at Array.<anonymous> (heatmap_chart.js:267)
    at Array.d3_selectionPrototype.call (d3.js:975)
    at point_series.js:231
    at arrayEach (index.js:1289)
    at Function.<anonymous> (index.js:3345)
```

Just internal error to console, no user-facing impact I believe